script/fix_readme_tags.py

```
@@ -0,0 +1,649 @@
+    out = {}
+    for rel in PDF_DIRS:
+        directory = ROOT / rel
+        if not directory.is_dir():
+            continue
+        for name in sorted(os.listdir(directory)):
+            if not name.lower().endswith(".pdf"):
+                continue
+            company = pdf_company(name)
+            if company not in CORE_COMPANIES:
+                continue
+            text = subprocess.run(["pdftotext", "-layout", str(directory / name), "-"],
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)